conans/requirements.txt

```diff
@@ -6,7 +6,7 @@ PyYAML>=3.11, <6.0
 patch-ng>=1.17.4, <1.18
 fasteners>=0.14.1
 six>=1.10.0,<=1.14.0
-node-semver>=0.8.0
+node-semver>=0.6.1, <0.8.0
```

Comment:
But if 0.8.0 is excluded we will not get the new feature?

Reply:
0.8.0 does not support python 2.7: https://github.com/conan-io/conan/pull/6941#issuecomment-624077161

However, the new feature it's the micro comparison that I moved to Version class, which means, we can continue with 0.6.1 and well have same behavior like 0.8.0